src/views/Course.vue

```diff
@@ -150,15 +150,50 @@
                       <h4 class="title">Recordings scheduled</h4>
-                      <div v-if="$currentUser.isAdmin">
-                        <a
-                          id="link-to-edit-kaltura-event"
-                          :href="`${$config.kalturaMediaSpaceUrl}/recscheduling/index/edit-event/eventid/${scheduled.kalturaScheduleId}`"
-                          target="_blank"
-                          aria-label="Open Kaltura MediaSpace in a new window"
-                        >
-                          Kaltura event {{ scheduled.kalturaScheduleId }} <v-icon small class="pb-1">mdi-open-in-new</v-icon>
-                        </a>
+                      <div v-if="$currentUser.isAdmin" class="d-flex align-bottom">
+                        <v-dialog v-if="$currentUser.isAdmin && scheduled.kalturaSchedule" v-model="kalturaScheduleDialogJSON">
+                          <template v-slot:activator="{ on }">
+                            <v-btn
+                              class="pa-0"
+                              text
+                              x-small
+                              v-on="on"
+                            >
+                              <v-icon id="kaltura-schedule-dialog" class="pb-2" v-on="on">
+                                mdi-code-json
+                              </v-icon>
+                            </v-btn>
+                          </template>
+                          <v-card>
+                            <v-card-title primary-title>
+                              Raw JSON of Kaltura schedule
+                            </v-card-title>
+                            <v-card-text>
+                              <pre>{{ scheduled.kalturaSchedule }}</pre>
+                            </v-card-text>
+                            <v-divider></v-divider>
+                            <v-card-actions>
+                              <v-spacer></v-spacer>
+                              <v-btn
+                                color="primary"
+                                text
+                                @click="kalturaScheduleDialogJSON = false"
+                              >
+                                Close
+                              </v-btn>
+                            </v-card-actions>
+                          </v-card>
+                        </v-dialog>
+                        <div>
+                          <a
+                            id="link-to-edit-kaltura-event"
+                            :href="`${$config.kalturaMediaSpaceUrl}/recscheduling/index/edit-event/eventid/${scheduled.kalturaScheduleId}`"
+                            target="_blank"
+                            aria-label="Open Kaltura MediaSpace in a new window"
+                          >
+                            Kaltura event {{ scheduled.kalturaScheduleId }} <v-icon small class="pb-1">mdi-open-in-new</v-icon>
+                          </a>
+                        </div>
```

Comment:
Schedule-related JSON from Kaltura, available to admins on `/course` page, is quite useful. Eg, you can grab `templateEntryId` and use API console to verify mapped categories (Canvas sites).